packages/components/src/components/block/block.tsx

```diff
@@ -210,7 +210,7 @@ export class Block extends LitElement implements InteractiveComponent {
   /**
    * Displays a status-related indicator icon.
    *
-   * @deprecated Use `icon-start` instead.
+   * @deprecated in v3.0, removal target v6.0 - Use `icon-start` instead.
```

Comment:
Consistency and wording suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 - Use the `iconStart` property instead.
```